python/setup.py

```diff
@@ -6,10 +6,25 @@
 
 setuptools.setup(
     name="spear-sim",
-    version="0.3.0",
+    version="0.5.0",
     author="",
     author_email="",
     description="",
     packages=["spear"],
-    install_requires=["ffmpeg-python==0.2.0", "gym==0.21.0", "matplotlib==3.6.2", "numpy==1.22.3", "opencv-python==4.5.5.64", "pandas==1.5.1", "psutil==5.9.0", "tensorflow==2.13.0", "yacs==0.1.8"]
-)
+    install_requires=[
+        "coacd==1.0.0"
```

Comment:
typo - there should be a `,` after `"coacd==1.0.0"`

Reply:
Good catch.